src/python/deepgnn/graph_engine/snark/tests/e2e_test.py

```diff
@@ -698,25 +768,25 @@ def test_feature_extraction_after_reset(multi_partition_graph_data):
         )
 
 
-@pytest.mark.parametrize("multi_partition_graph_data", param, indirect=True)
+@pytest.mark.parametrize("multi_partition_graph_data", ["original"], indirect=True)
```

Comment:
Why is param switched to just ["original"] shouldn't this test be run on all input formats?

Reply:
It doesn't look like it was tested at all. I noticed there were some failures reported, but overall `bazel test` command printed green. It turns out python fixtures were failing and, in this case, `pytest` will not report a test failure, but rather test to be in error state. The culprit was a crash in starting a GE to test:
<img width="1032" alt="image" src="https://user-images.githubusercontent.com/134559/198699911-1ddef056-8a0b-4d44-80eb-8e25df819b6b.png">
And from what I can tell these tests were [failing since they were introduced](https://github.com/microsoft/DeepGNN/actions/runs/3340633799/jobs/5531092167):
<img width="686" alt="image" src="https://user-images.githubusercontent.com/134559/198700504-bb3e0059-777e-46f5-a00b-7fcc24e85752.png">

Hence, I made them to run on parameters they are passing, because it doesn't look like they should run with other premutation.

Reply:
I see, should this be added as a task to fix or are they inconsequential?

Reply:
Graph binaries should be the same in e2e tests and most of other tests in this module already have a filter for `["original"]` parameters, so it seems like these tests should've had this filter anyway.

Reply:
Tests are fixed now in a separate PR and everything is passing here.